### PR TITLE
Add ArangoDB driver

### PR DESCRIPTION
**Pull request by sandrokeil**

### CODE REVIEW

File: src/functions.php [hunk 1]
```diff
@@ -19,13 +27,17 @@
...
+use Prooph\EventStore\ArangoDb\EventStore as ArangoDbEventStore;
+use Prooph\EventStore\ArangoDb\PersistenceStrategy\AggregateStreamStrategy as ArangoDbAggregateStreamStrategy;
 
 function testDatabases(): array
 {
     return [
         'mysql' => getenv('MYSQL_DB'),
         'mariadb' => getenv('MARIADB_DB'),
         'postgres' => getenv('POSTGRES_DB'),
+        'arangodb' => getenv('ARANGODB_DBNAME'),
```
Comment (prolic) [inline src/functions.php]: I used DB for all other drivers, not DBNAME. Let's make this consistent, either DB or DBNAME, you can chose :)

Comment (sandrokeil) [reply src/functions.php]: Done

**Comment by sandrokeil** [comment]
@prolic done

**Comment by prolic** [comment]
Sorry I don't have the time today, but hopefully Tuesday or Wednesday.

**Pull request merged**